.gitignore

```diff
@@ -8,7 +8,11 @@ Include
 bin
 gen
 obj
-libs
+**/libs/eegeo/
```

Comment:
I think this is all redundant -- "libs" (as it was) should cover recursive directory content.